mlflow/langchain/__init__.py

```diff
@@ -961,6 +962,7 @@ def autolog(
     silent=False,
     registered_model_name=None,
     extra_tags=None,
+    extra_log_classes=None,
```

Comment:
🎏  Not sure if this is useful

Reply:
I'd remove this unless someone requests it.

Reply:
`extra_model_classes`